### DISCUSSION TITLE
Streaming Legacy Mode

### DISCUSSION THREAD

**Discussion by crossic84** [dgreif/ring#1268] [Q&A]
Hello, since the last firmware update Doorbell stopped to stream, I have read [https://github.com/dgreif/ring/wiki/Streaming-Legacy-Mode](https://github.com/dgreif/ring/wiki/Streaming-Legacy-Mode) but didnt find a way how to change it, was anyone else lucky to fing the settings hot to change it back to H264?

**Comment by smockle** [comment]
From https://github.com/dgreif/ring/issues/1253#issuecomment-1642481844:
> Log in to the Ring web dashbaord using Firefox. It must be Firefox, because that is the only browser that does not support **any** playback of HEVC files, even via direct download (most browsers have some support for HEVC, just not via WebRTC which is used for live viewing). Go to your event history for this camera and select an event. It will not be able to display, and you will get a popup that allows switching to Legacy mode.

**Reply by smockle** [reply]
Incidentally, a later comment (https://github.com/dgreif/ring/issues/1253#issuecomment-1642481844) suggests that, eventually, switching to Legacy Mode will be unnecessary:
> …I thought the best short-term option was going to be [setting] the camera to legacy mode, however, something interesting happened this week: Ring changed the way they stream for web browsers! …Overall this is not particularly interesting except, when the API is used with this slightly different authentication, it suddenly does allow negotiating down to H.264 for WebRTC! …[T]his API appears to work with any camera, Ring Edge or not…and allows HEVC cameras to be negotiated down to H.264/AVC on-the-fly, which is the best of all possible worlds.

**Comment by crossic84** [comment]
> From [#1253 (comment)](https://github.com/dgreif/ring/issues/1253#issuecomment-1642481844):
> 
> > Log in to the Ring web dashbaord using Firefox. It must be Firefox, because that is the only browser that does not support **any** playback of HEVC files, even via direct download (most browsers have some support for HEVC, just not via WebRTC which is used for live viewing). Go to your event history for this camera and select an event. It will not be able to display, and you will get a popup that allows switching to Legacy mode.

Sound great man, thx so much, i do not have a protect plan so I have no events to be displayed there :X
